src/OpenTracing/Formats.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace OpenTracing\Formats;
+
+/**
+ * Used a (single) arbitrary binary blob representing a SpanContext
+ */
+const BINARY = 'binary';
+
+/**
+ * Used for an arbitrary string-to-string map with an unrestricted character set for both keys and values
+ *
+ * Unlike `HTTP_HEADERS`, the `TEXT_MAP` format does not restrict the key or
+ * value character sets in any way.
+ *
+ * For both Tracer::inject() and Tracer::extract() the carrier must be a `array|ArrayObject`.
```

Comment:
just for my info, is array/ArrayObject in PHP an associative array (like in Javascript)?

Reply:
Every array in PHP is associative, _unlike_ Javascript

Reply:
Thanks. I meant "like in Javascript objects"